module/VuFindSearch/src/VuFindSearch/Response/SimpleRecord.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * A minimal record class for wrapping an array of fields
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Sitemap
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+namespace VuFindSearch\Response;
+
+/**
+ * A minimal record class for wrapping an array of fields
+ *
+ * @category VuFind
+ * @package  Sitemap
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+class SimpleRecord implements RecordInterface
+{
+    /**
+     * Source backend identifier
+     */
+    protected $sourceId = 'Solr';
```

Comment:
Any objection to using DEFAULT_SEARCH_BACKEND here in place of the hard-coded Solr?

Reply:
No objection, fixed.